Process.c

```diff
@@ -1274,13 +1277,14 @@ void Process_updateFieldWidth(ProcessField key, size_t width) {
 }
 
 void Process_updateCPUFieldWidths(float percentage) {
-   if (percentage < 99.9) {
+   if (percentage < 99.9F) {
       Process_updateFieldWidth(PERCENT_CPU, 4);
       Process_updateFieldWidth(PERCENT_NORM_CPU, 4);
       return;
    }
 
-   uint8_t width = ceil(log10(percentage + .2));
+   // Add additional two characters, one for "." and other for precision.
+   uint8_t width = ceil(log10(percentage + 0.1)) + 2;
```

Comment:
I'm sorry that I made an imprecise algorithm in my last patch.
Here I make a better one. I avoid the 0.1 value in the addition as 0.1 doesn't have exact representation in binary floating point. I multiply the value by 10 instead (which is exact).

```suggestion
   // The width includes a decimal point and one fraction digit.
   uint8_t width = (int)ceil(log10(percentage * 10.0 + 1.0)) + 1;
```

Reply:
The point of the `+0.1` is less about precision, but about over-estimating the needed number of digits.